$NetBSD: patch-ae,v 1.4 2018/02/19 21:25:46 triaxx Exp $

Fix build on OpenBSD and Bitrig.
Calling usleep for 1000000 microseconds or more is not allowed, so add a
myusleep() wrapper to DTRT.

--- src/proxy.h.orig	2017-10-19 14:25:44.000000000 +0000
+++ src/proxy.h
@@ -21,7 +21,9 @@
 #include <ctype.h>
 #include <sys/types.h>
 #include <sys/stat.h>
+#if !(defined(__OpenBSD__) || !defined(__Bitrig__))
 #include <sys/timeb.h>
+#endif
 #include <fcntl.h>
 #include <time.h>
 
@@ -107,6 +109,8 @@
 #endif
 void daemonize(void);
 #define SLEEPTIME 1000
+#define usleep(usecs) myusleep(usecs)
+int myusleep(useconds_t);
 #ifndef O_BINARY
 #define O_BINARY 0
 #endif
